en/flight_controller/pixhawk4.md

```diff
@@ -1,8 +1,8 @@
 # Pixhawk 4 
 
```

Comment:
A lot of the changes in this doc are:
- Order of things - assembly for example is towards the end, purchase up the top. Reasoning being that most newbies will have got here from the quickstart guide. 
- I always refer to the pins by their pin name as printed on front. So **POWER1** not Power Module 1 (or whatever)